package.json

```diff
@@ -127,5 +127,6 @@
       }
     },
     "includedScripts": []
-  }
+  },
+  "packageManager": "yarn@1.22.21+sha1.1959a18351b811cdeedbd484a8f86c3cc3bbaf72"
```

Comment:
Please revert this, it shouldn't have been added in the first place, but I have also migrated from yarn classic to pnpm in the meantime and this now prevents CI from running

Reply:
This was marked as resolved before it was addressed?

Reply:
Yeah, I was marking things off as I made the changes locally. I've just pushed up all of the changes.